Update VM Details - Clarification around OU format (#59175)
Azure/azure-content: articles/virtual-desktop/create-host-pools-azure-marketplace.md

10. After that, select whether you want the virtual machines to be joined to a specific domain and organizational unit. If you choose **Yes**, specify the domain to join. You can also add a specific organizational unit you want the virtual machines to be in.

    >[!NOTE]

>     >**DO NOT** enter the Active Directory Organizational Unit Distinguished Name in quotations. If entered in quotations, the domain join deployment activity will fail. 

Reviewer: ```suggestion
    >Don't enter the Active Directory Organizational Unit Distinguished Name in quotations. If entered in quotations, the domain join won't work. 
```

Reviewer: Actually, "Don't enter the Active Directory Unit Distinguished Name with quotation marks around it. If you enter the name with quotation marks, the domain join won't work."
